### ISSUE TITLE
GraphQL errors after installation on type "MdxFields"

### ISSUE THREAD

**Issue by davidlevy** [mrmartineau/gatsby-theme-code-notes#38]
Hi

After running

`gatsby new code-notes https://github.com/MrMartineau/gatsby-starter-code-notes
cd code-notes
yarn dev`

I got these blocking errors in the CLI : 

Cannot query field "dateModified" on type "MdxFields".
The fragment "AllPages" does not exist.

I got : 
```
% yarn --version
1.3.2
% node --version
v12.5.0
```

Did I miss something ? 
Thanks !

**Comment by mrmartineau** [comment]
Hi @davidlevy, I tried to reproduce your error but wasn't able to. Could you try again and see if you get the same errors? 

The `dateModified` information comes from the git modified date which, when using the `gatsby new` command, should be there. On the flip side, this information is not currently being used, so I can remove it until I have a working implementation. Issue #11 has more information on this.

**Comment by techieshark** [comment]
Same here. 

I run:
```
mkdir my-site
cd my-site
yarn init

# install gatsby-theme-code-notes and it's dependencies
yarn add gatsby-theme-code-notes gatsby react react-dom
```
and get:
```
There was an error in your GraphQL query:

Expected type MdxFieldsEnum, found fields___dateModified.

File: /...code-notes/node_modules/gatsby-theme-code-notes/src/templates/Notes.js

There was an error in your GraphQL query:

Cannot query field "dateModified" on type "MdxFields".

If you don't expect "dateModified" to exist on the type "MdxFields" it is most likely a typo.
However, if you expect "dateModified" to exist there are a couple of solutions to common problems:

- If you added a new data source and/or changed something inside gatsby-node.js/gatsby-config.js, please try a restart of your development server
- The field might be accessible in another subfield, please try your query in GraphiQL and use the GraphiQL explorer to see which fields you can query and what shape they have
- You want to optionally use your field "dateModified" and right now it is not used anywhere. Therefore Gatsby can't infer the type and add it to the GraphQL schema. A quick fix is to add a least one entry with that field ("dummy content")

It is recommended to explicitly type your GraphQL schema if you want to use optional fields. This way you don't have to add the mentioned "dummy content". Visit our docs to learn how you can define the schema for "MdxFields":
https://www.gatsbyjs.org/docs/schema-customization/#creating-type-definitions
```

perhaps because I haven't commited to git yet?

... update: Yep. If I `git init && git add . && gatsby clean && gatsby dev` it works.

I noticed the same error above if I used the starter project and renamed the `example-note.md` file (apparently it causes issues due to the reliance on git for the date modification timestamp?).

**Comment by mrmartineau** [comment]
It seems that we need to remove the git timestamp stuff for now. I don't have time to look at this right now so if it is important to you, please submit a PR for it. 

There are a few things that need to be done:
- remove this line: https://github.com/mrmartineau/gatsby-theme-code-notes/blob/master/gatsby-theme-code-notes/gatsby-node.js#L6
- remove these lines: https://github.com/mrmartineau/gatsby-theme-code-notes/blob/master/gatsby-theme-code-notes/gatsby-node.js#L166-L175
- and anywhere else that mentions [`dateModified`](https://github.com/mrmartineau/gatsby-theme-code-notes/search?q=dateModified&unscoped_q=dateModified)

Hope that helps 😄

**Comment by mrmartineau** [comment]
FYI this is being fixed by #51
